requirements.txt

```diff
@@ -0,0 +1,3 @@
+esphome>=2023.2.0
+pre-commit>=3.4.0
+yamllint>=1.32.0
```

Comment:
This was added in pre-commit so no reason to additionally require it.

Reply:
Done.